crates/rpc/rpc-builder/src/auth.rs

```diff
@@ -375,7 +380,7 @@ pub struct AuthServerHandle {
     local_addr: SocketAddr,
     handle: ServerHandle,
     secret: JwtSecret,
-    _ipc_handle: ServerHandle,
+    _ipc_handle: Option<ServerHandle>,
```

Comment:
we can add a public function that returns the Ipc Handle so this is no longer unused

Reply:
hack, got it